http/src/http-driver.ts

```diff
@@ -117,8 +118,8 @@ export function createResponse$(reqInput: RequestInput): Stream<Response> {
       }
     },
     stop: function stopResponseStream() {
-      if (this.request && this.request.abort) {
-        this.request.abort();
+      if (request && request.abort) {
+        request.abort();
```

Comment:
I just noticed that in both the `this` implementation as with this new closure `request` implementation, we should reset the request back to undefined or null in this `stop()` method, so that the next execution of this stream doesn't accidentally call "abort" on the request of the previous execution. It's a rare corner case that may not happen, but since we're at it, might be good to add `request = null` just in case.